PSKoans/Koans/Constructs and Patterns/AboutSplatting.Koans.ps1

```diff
@@ -77,13 +80,23 @@ Describe 'Splatting' {
 
         It 'can be built from automatic hashtables' {
             $String = "Folders:__"
-            $Parameters = if ($String -match 'Folders:(?<Filter>[a-z ])$') {
+            $Parameters = if ($String -match 'Folders:(?<Filter>[_\d\w ]*)$') {
+                <#Matches is automaticed populated by the -match operater in the if statement
+                If $String = "Folders:00_TheBasics"
+                Then $Matches holds the following:
+                Name                           Value
+                ----                           -----
+                Filter                         00_TheBasics
+                0                              Folders:00_TheBasics
+                #>
```

Comment:
Can we reformat this to match the block-comment style used elsewhere in the koans? Usually I keep the opening/closing lines clear and indent the comments, like so:

```powershell
<#
    Comments here.
#>
```